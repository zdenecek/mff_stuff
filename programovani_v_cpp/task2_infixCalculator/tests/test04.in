5
6 % 3
5 % 2
1 % 1000
0 % 0
5.5 % 3