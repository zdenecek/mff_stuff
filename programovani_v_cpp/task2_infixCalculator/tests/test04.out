0
1
1
invalid
2.5
